src/libraries/System.Private.CoreLib/src/System/Byte.cs

```diff
@@ -23,7 +23,8 @@ namespace System
           IMinMaxValue<byte>,
           IUnsignedNumber<byte>,
           IUtf8SpanFormattable,
-          IUtfChar<byte>
+          IUtfChar<byte>,
+          IBinaryIntegerParseAndFormatInfo<byte>
```

Comment:
Should we fold IUtfChar into IBinaryIntegerParseAndFormatInfo and then delete the former?

Reply:
I opted not to since its only implemented for the former two. We can always revisit later if desired.